### PR TITLE
Updated motion planning panel

### PR DESCRIPTION
**Pull request by yimingyanged**
- Added bi-hands reaching option
- Added walking plan fake execute option
- Some UI updates

### CODE REVIEW

File: src/python/director/motionplanningpanel.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def __init__(self, robotStateModel, robotStateJointController, teleopRobotModel,
         # Foot step planning
         self.placer = None
         self.ui.walkingPlanButton.connect('clicked()', self.onWalkingPlan)
-        
+        self.ui.walkingFakeExeButton.connect('clicked()', self.onWalkingFakeExecute)
```
Comment (mauricefallon) [inline src/python/director/motionplanningpanel.py]: what is "fake exe"

**Comment by mauricefallon** [comment]
seems reasonable but we use "teleport" to mean "moving robot model in the UI by not in reality"

File: src/python/director/motionplanningpanel.py (hunk 1 above)
Comment (yimingyanged) [inline src/python/director/motionplanningpanel.py]: 'fake exe' simply move the robot to the desired location without actual 'walking'. It provides a way to evaluate and test final pose planning and motion planning in a much simpler manner

Comment (yimingyanged) [inline src/python/director/motionplanningpanel.py]: It is similar to MoveIt's fake execution (where I got the name from)

Comment (mauricefallon) [inline src/python/director/motionplanningpanel.py]: i understand, please change to teleportRobotToStanceFrame can see if you can make it identical to the version in tabledemo.py and valvedemo.py

where we have common functions like this we often move them to robottasks.py

**Pull request merged**